.github/actions/get-tools-versions/action.yml

```diff
@@ -22,69 +22,64 @@ outputs:
   tlf: 
     description: "TLF-Library R package version"
     value: ${{ steps.tools-versions.outputs.tlf  }}
+  r: 
+    description: "R version"
+    value: ${{ steps.tools-versions.outputs.r  }}
   tools-table:
     description: "Markdown table corresponding to csv tools information"
     value: ${{ steps.tools-table.outputs.table  }}
   
 runs:
   using: "composite"
   steps:
-    - id: install-r
-      name: Setup R
-      uses: r-lib/actions/setup-r@v2
-      with:
-        r-version: '4.3.1'
-        use-public-rspm: true
-        
-    - id: check-tools
-      name: Check tools input
+    - id: tools-versions
+      name: Export versions of certain tools
       run: |
-        install.packages(c("jsonlite"), repos = 'http://cran.us.r-project.org')
-        toolsData <- read.csv("${{ inputs.tools-path }}", stringsAsFactors = FALSE, colClasses = "character")
-        toolsData <- sapply(
-          toolsData$Tool, 
-          function(toolName){
-            list(
-              version = toolsData$Version[toolsData$Tool %in% toolName], 
-              url = toolsData$URL[toolsData$Tool %in% toolName]
-            )
-          }, 
-          USE.NAMES = TRUE,
-          simplify = FALSE
-          )
-        # Save PK sim and Qualifcation framework versions
-        cat(toolsData$`PK-Sim`$version, file = "pk-sim-version.txt")
-        cat(toolsData$`Qualification Framework`$version, file = "qualification-framework-version.txt")
-        cat(toolsData$`ospsuite-R`$version, file = "ospsuite-r-version.txt")
-        cat(toolsData$`Reporting Engine`$version, file = "reporting-engine-version.txt")
-        cat(toolsData$TLF$version, file = "tlf-version.txt")
-      shell: Rscript {0}
-    
+        # Default R version
+        echo "r=latest" >> $GITHUB_OUTPUT
+        # Process each line after header assumes columns are Name, Version and URL
+        tail -n +2 "${{ inputs.tools-path }}" | while IFS=',' read -r val1 val2 val3
+        do
+          # Save versions corresponding to appropriate name
+          if [[ "$val1" == "PK-Sim" ]]; then
+            echo "pk-sim=$val2" >> $GITHUB_OUTPUT
+          fi
+          if [[ "$val1" == "Qualification Framework" ]]; then
+            echo "qualification-framework=$val2" >> $GITHUB_OUTPUT
+          fi
+          if [[ "$val1" == "ospsuite-R" ]]; then
+            echo "ospsuite-r=$val2" >> $GITHUB_OUTPUT
+          fi
+          if [[ "$val1" == "Reporting Engine" ]]; then
+            echo "reporting-engine=$val2" >> $GITHUB_OUTPUT
+          fi
+          if [[ "$val1" == "TLF" ]]; then
+            echo "tlf=$val2" >> $GITHUB_OUTPUT
+          fi
+          if [[ "$val1" == "R" ]]; then
+            echo "r=$val2" >> $GITHUB_OUTPUT
+          fi
+        done
+      shell: bash
```

Comment:
@pchelle 
why are some tools (RUtils, rSharp, Qualification Runner) not processed here?